tests/integration/transactions/test_account_delete.py

```diff
@@ -38,3 +39,17 @@ def test_all_fields(self):
         # Sequence number that is too high. The current ledger index must be at least
         # 256 higher than the account's sequence number."
         # self.assertEqual(response.result['engine_result'], 'tesSUCCESS')
+
+    def test_high_fee_unauthorized(self):
+        # We expect an XRPLException to be raised
+        with self.assertRaises(XRPLException):
+            # GIVEN a new AccountDelete transaction
+            account_delete = AccountDelete(
+                account=ACCOUNT,
+                # WITH fee higher than 2 XRP
+                fee=FEE,
+                sequence=WALLET.sequence,
+                destination=DESTINATION.classic_address,
+                destination_tag=DESTINATION_TAG,
+            )
+            submit_transaction(account_delete, WALLET)
```

Comment:
This test should be in `tests/integration/sugar/test_transaction`, because it's testing transaction sugar

Reply:
I moved it.